CourseGrab/ViewControllers/Search/SearchTableViewCell.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  SearchTableViewCell.swift
+//  CourseGrab
+//
+//  Created by Reade Plunkett on 2/4/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class SearchTableViewCell: UITableViewCell {
+
+    private let arrowImageView = UIImageView()
+    private let divider = UIView()
+    private let containerView = UIView()
+    private let statusBadge = UIImageView()
+    private let subtitleLabel = UILabel()
+    private let titleLabel = UILabel()
+    private let trackingButton = UIButton()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        // Setup appearance
+
+        selectionStyle = .none
+
+        arrowImageView.backgroundColor = .black
+        arrowImageView.contentMode = .scaleAspectFit
+        containerView.addSubview(arrowImageView)
+
+        divider.backgroundColor = .courseGrabBorder
+        containerView.addSubview(divider)
+        divider.isHidden = true
+
+        containerView.clipsToBounds = false
+        containerView.layer.cornerRadius = 5
+        containerView.layer.shadowColor = UIColor.black.cgColor
+        containerView.layer.shadowOpacity = 0.3
+        containerView.layer.shadowRadius = 2
+        containerView.layer.shadowOffset = .zero
+        containerView.backgroundColor = .white
+        contentView.addSubview(containerView)
+
+        titleLabel.font = ._16Semibold
+        titleLabel.numberOfLines = 0
+        containerView.addSubview(titleLabel)
+
+        trackingButton.isHidden = true
+        trackingButton.layer.cornerRadius = 2
+        trackingButton.layer.borderWidth = 1
+        trackingButton.layer.borderColor = UIColor.courseGrabRuby.cgColor
+        trackingButton.titleLabel?.font = ._12Medium
+        trackingButton.setTitle("REMOVE", for: .normal)
+        trackingButton.setTitleColor(.courseGrabRuby, for: .normal)
+        trackingButton.layer.borderColor = UIColor.courseGrabRuby.cgColor
+        containerView.addSubview(trackingButton)
+
+        statusBadge.contentMode = .scaleAspectFit
+        containerView.addSubview(statusBadge)
+        statusBadge.isHidden = true
+
+        subtitleLabel.font = ._14Semibold
+        containerView.addSubview(subtitleLabel)
+        subtitleLabel.isHidden = true
```

Comment:
For line 60 too, I would say `.isHidden = true` before `addSubview` just so that, for the sake of consistency, we do appearance, then `addSubview`, then constraints.